tests/api/data/alerts/active__status=actual&area=AL.json

```diff
@@ -19,127 +19,10 @@
         "id": "urn:oid:2.49.0.1.840.0.21973bd423ae0db39e5260d08fa31bb6eba8b288.001.1",
         "areaDesc": "Stewart; Montgomery; Robertson; Sumner; Macon; Clay; Pickett; Houston; Humphreys; Dickson; Cheatham; Davidson; Wilson; Trousdale; Smith; Jackson; Putnam; Overton; Fentress; Perry; Hickman; Lewis; Williamson; Maury; Marshall; Rutherford; Cannon; De Kalb; White; Cumberland; Bedford; Coffee; Warren; Grundy; Van Buren; Wayne; Lawrence; Giles",
         "geocode": {
-          "SAME": [
```

Comment:
For all these saved data files, I had to make a few changes:

1. Because we now get alerts based on state, I needed to change the filename from `&point=[lat,lon]` to `&area=[state]`. So I also had to update the `gridpoints/0,0.json` (etc) files to have real coordinates that would resolve to real places. I just started doing states in alphabetical order. (I skipped Alaska because it felt rude to use a state as a test case when we can't actually show anything for large swaths of it.)
2. We also don't just accept the data we get. We apply additional filtering on top of it, so we need to make sure the alerts will pass. Rather than trying to get them to fit a polygon, I opted to just update the SAME codes for all of them.